packages/vats/src/bridge.js

```diff
@@ -61,12 +63,14 @@ const prepareScopedManager = zone => {
         assert(inboundHandler, X`No inbound handler for ${bridgeId}`);
         return E(inboundHandler).fromBridge(obj);
       },
-      setHandler(newHandler) {
+      setHandler(newHandler, oldHandler) {
```

Comment:
We should not require an `oldHandler` argument, as discussed internally.  Can you please revert this addition?